### PR TITLE
Added checksum check when downloading plugin from Cosmos.

### PR DESCRIPTION
**Pull request by ArmandGrillet**

### CODE REVIEW

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

File: pkg/plugin/manager.go [hunk 1]
```diff
@@ -258,9 +270,23 @@ func (m *Manager) downloadPlugin(url string) (string, error) {
...
+		respReader = resp.Body
+	}
+	if err := fsutil.CopyReader(m.fs, respReader, downloadedFilePath, 0644); err != nil {
 		return "", err
 	}
+
+	if checksum.Hasher != nil {
+		m.logger.Debugf("Verifying checksum for %s...", url)
+		if hex.EncodeToString(checksum.Hasher.Sum(nil)) != checksum.Value {
+			return "", fmt.Errorf("computed checksum %s for %s, expected %s", hex.EncodeToString(checksum.Hasher.Sum(nil)), url, checksum.Value)
```
Comment (bamarni) [inline pkg/plugin/manager.go]: This will compute a second time in case of error, it'd be a bit better to store the result in a var.

**Comment by bamarni** [comment]
@mesosphere-ci run integration tests

**Comment by ArmandGrillet** [comment]
@mesosphere-ci run integration tests

**Pull request merged**